jsx.js

```diff
@@ -13,7 +13,78 @@ module.exports = {
   },
 
   rules: {
-    'jsx-quotes': [2, 'prefer-double']
+    'jsx-quotes': [2, 'prefer-double'],
+    'react/boolean-prop-naming': [0],
+    'react/button-has-type': [0],
+    'react/default-props-match-prop-types': [2],
+    'react/destructuring-assignment': [0],
+    'react/forbid-component-props': [0],
+    'react/forbid-dom-props': [0],
+    'react/forbid-elements': [0],
+    'react/forbid-prop-types': [2],
+    'react/function-component-definition': [2, {namedComponents: 'arrow-function'}],
+    'react/jsx-boolean-value': [0],
+    'react/jsx-child-element-spacing': [2],
+    'react/jsx-closing-bracket-location': [2, 'line-aligned'],
+    'react/jsx-closing-tag-location': [2],
+    'react/jsx-curly-brace-presence': [2, {props: 'never', children: 'never'}],
+    'react/jsx-curly-newline': [2],
+    'react/jsx-curly-spacing': [2, 'never'],
+    'react/jsx-equals-spacing': [0],
+    'react/jsx-filename-extension': [0],
+    'react/jsx-first-prop-new-line': [2],
+    'react/jsx-fragments': [2, 'syntax'],
+    'react/jsx-handler-names': [0],
+    'react/jsx-indent': [2, 2, {checkAttributes: true, indentLogicalExpressions: true}],
+    'react/jsx-indent-props': [2, 2],
+    'react/jsx-max-depth': [0],
+    'react/jsx-max-props-per-line': [2],
+    'react/jsx-newline': [2, {prevent: true}],
+    'react/jsx-no-constructed-context-values': [2],
+    'react/jsx-no-literals': [0],
+    'react/jsx-no-script-url': [0],
+    'react/jsx-no-useless-fragment': [2, {allowExpressions: true}],
+    'react/jsx-one-expression-per-line': [2, {allow: 'literal'}],
```

Comment:
I'd prefer `{allow: 'single-child'}` as configuration for this rule, again not to have a lot of very short lines.

Reply:
👍